examples/argocd-openshift.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: argocd-operator
+  namespace: argocd
```

Comment:
I had issues creating this in the argocd namespace.
- the namespace needs to exist first. Simple fix of creating (which would negate the need to create in the cmd in the other doc `kubectl create ns argocd && kubectl apply -f manifests/argocd-install/app-of-apps-application.yaml `
- the CRDs didn't exist yet, so only the subscription is created first time
- the operator never got deployed, and the subscription was removed automatically. This is probably an environment issue unrelated to the instructions & yaml here. However, if we change this to the openshift-operators namespace, I think that might work better (as that's what got created when I went through the openshift UI)

Reply:
I have tested this and the installation only works in the openshift-operators namespace. 
I have also split it in two files to avoid the failure due to the subscription needing to be installed first. I thought about providing a make target to install it, but there are way too many options when installing argocd and I think that's out of the scope of this repo. An example is more than enough for someone to get started.

56e15fb